installer.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # Increase this version number whenever you update the installer
 INSTALLER_VERSION="2020-06-12" # format YYYY-MM-DD
```

Comment:
We also need to adjust the version here!

Reply:
Thanks a lot for your review I increased the version for the installer and added a changelog entry.
I think putting it into the Linux changelog should be ok?